docs/4.2/ssh_azuread.md

```diff
@@ -0,0 +1,190 @@
+# SSH Authentication with Azure Active Directory (AD)
+
+This guide will cover how to configure [Microsoft Azure Active Directory(https://azure.microsoft.com/en-us/services/active-directory/) to issue
+SSH credentials to specific groups of users with a SAML Authentication Connector. When used in combination with role
+based access control (RBAC) it allows SSH administrators to define policies
+like:
+
+* Only members of "DBA" Google group can SSH into machines running PostgreSQL.
+* Developers must never SSH into production servers.
+* ... and many others.
+
+The following steps configure an example SAML authentication connector matching AzureAD groups with security roles.  You can choose to configure other options.
+
+!!! warning "Version Warning"
+
+    This guide requires an enterprise version of Teleport. The open source
+    edition of Teleport only supports [Github](admin-guide.md#github-oauth-20) as
+    an SSO provider.
+
+## Prerequisites:
+
+Before you get started you’ll need:
+
+- An Enterprise version of Teleport v4.2 or greater, downloaded from [https://dashboard.gravitational.com/](https://dashboard.gravitational.com/web/). 
+- Azure AD admin account with access to creating non-gallery applications (P2 License)
+- Register one or more users in the directory
```

Comment:
Register -> To register (because of the "you'll need" header)

Reply:
changed